sdcm/prometheus.py

```diff
@@ -4,22 +4,32 @@
 import logging
 import prometheus_client
 
+from avocado.utils import network
+
 START = 'start'
 STOP = 'stop'
 
 logger = logging.getLogger(__name__)
 nm_obj = None
 
 
-def start_metrics_server():
-    seed = os.getpid() / 100
-    port = random.randint(8000 + seed, 10000 - seed)
+def start_metrics_server(port=9389):
+    """
+    https://github.com/prometheus/prometheus/wiki/Default-port-allocations
+    Occupied port 9389 for SCT
+    """
+    hostname = socket.gethostname()
+    if not network.is_port_free(port, hostname):
+        port = network.find_free_port(8001, 10000)
+
```

Comment:
We cannot use the same port for different loaders. To provide different ports the random seed was used. 
Please add it back. We can generate it and check if it's free until we get the free one.

Reply:
Then I know the purpose of getpid & seed.

1. current start_metrics_server() just tries to start a new prometheus API server in new port, it doesn't restart (stop & restart) existing one.
2. we don't need to restart prometheus API server, which is running by a daemon thread. The server can be reused directly.
3. this PR works as past. In second call of start_metrics_server(), the original port isn't free, it will start server from new port (as past code)

Reply:
I see we don't restart the HTTP server on monitor, we are using the same port on reconfigure and restart of prometheus service. So I'm wrong - we can use the default port :)